Source/WebCore/dom/ActiveDOMObject.cpp

```diff
@@ -79,8 +79,7 @@ ActiveDOMObject::~ActiveDOMObject()
     // inherit). Hence, we should ensure that this is not 0 before use it
     // here.
 
-    // Unable to ref the context as it may have started destruction.
-    auto* context = scriptExecutionContext();
+    RefPtrAllowingPartiallyDestroyed<ScriptExecutionContext> context = scriptExecutionContext();
```

Comment:
Sadly, the compiler is forcing me to provide the template parameter for these new smart pointers, apparently because they're aliases:
```
Source/WebCore/dom/ActiveDOMObject.cpp:82:38: error: alias template 'RefPtrAllowingPartiallyDestroyed' requires template arguments; argument deduction only allowed for class templates
    RefPtrAllowingPartiallyDestroyed context = scriptExecutionContext();
```